### ISSUE TITLE
Cannot find symbols, not within bounds and cannot be instantiated Java building errors.

### ISSUE THREAD

**Issue by TWSICommunity** [greyhavens/msoy#21]
whirledtest@whirledtest-VirtualBox:~/Desktop/msoy$ ant distall
Buildfile: /home/whirledtest/Desktop/msoy/build.xml

config:
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist
     [copy] Copying 8 files to /home/whirledtest/Desktop/msoy/dist

prepare:
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist/lib
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist/build
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist/tmp
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist/classes
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist/classes/rsrc
     [copy] Copying 266 files to /home/whirledtest/Desktop/msoy/dist/classes/rsrc
    [mkdir] Created dir: /home/whirledtest/Desktop/msoy/dist/test-classes
[copyfilelist] Copied 28 files to /home/whirledtest/Desktop/msoy/dist/lib
     [copy] Copying 1 file to /home/whirledtest/Desktop/msoy/dist
     [copy] Copying 1 file to /home/whirledtest/Desktop/msoy/pages
     [copy] Copying 1 file to /home/whirledtest/Desktop/msoy/pages
     [copy] Copying 1 file to /home/whirledtest/Desktop/msoy/dist

copythanebinaries:

-download-maven-ant:

-init-maven-ant:
Trying to override old definition of task urn:maven-artifact-ant:dependencies
Trying to override old definition of task urn:maven-artifact-ant:install
Trying to override old definition of task urn:maven-artifact-ant:deploy
Trying to override old definition of task urn:maven-artifact-ant:install-provider
Trying to override old definition of task urn:maven-artifact-ant:mvn
Trying to override old definition of task urn:maven-artifact-ant:writepom
Trying to override old definition of datatype urn:maven-artifact-ant:localRepository
Trying to override old definition of datatype urn:maven-artifact-ant:remoteRepository
Trying to override old definition of datatype urn:maven-artifact-ant:authentication
Trying to override old definition of datatype urn:maven-artifact-ant:proxy
Trying to override old definition of task urn:maven-artifact-ant:pom
[artifact:dependencies] There were no dependencies specified

-prepare:

detect-arch:
     [echo] Architecture is Linux-32

distbinary:
     [copy] Copying 1 file to /home/whirledtest/Desktop/msoy/dist/lib

mavendeps:
[artifact:dependencies] Downloading: com/threerings/narya/1.4/narya-1.4.pom from repository central at http://repo1.maven.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:narya:pom:1.4' in repository central (http://repo1.maven.org/maven2)
[artifact:dependencies] [INFO] snapshot com.threerings:whirled-code:1.1-SNAPSHOT: checking for updates from ooo
[artifact:dependencies] [WARNING] repository metadata for: 'snapshot com.threerings:whirled-code:1.1-SNAPSHOT' could not be retrieved from repository: ooo due to an error: Error transferring file: build.earth.threerings.net
[artifact:dependencies] [INFO] Repository 'ooo' will be blacklisted
[artifact:dependencies] Downloading: com/threerings/whirled-code/1.0/whirled-code-1.0.pom from repository ooo-repo at http://build.earth.threerings.net/maven-repository
[artifact:dependencies] Error transferring file: build.earth.threerings.net
[artifact:dependencies] [WARNING] Unable to get resource 'com.threerings:whirled-code:pom:1.0' from repository ooo-repo (http://build.earth.threerings.net/maven-repository): Error transferring file: build.earth.threerings.net
[artifact:dependencies] Downloading: com/threerings/whirled-code/1.0/whirled-code-1.0.pom from repository central at http://repo1.maven.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirled-code:pom:1.0' in repository central (http://repo1.maven.org/maven2)
     [copy] Copying 79 files to /home/whirledtest/Desktop/msoy/dist/lib
     [copy] Copying 27 files to /home/whirledtest/Desktop/msoy/dist/build

mavendeps-as:
[artifact:dependencies] Downloading: org/osflash/signals/0.8/signals-0.8.pom from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'org.osflash:signals:pom:0.8' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: org/osflash/signals/0.8/signals-0.8.pom from repository ooo-repo at http://threerings.github.com/maven-repo
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'org.osflash:signals:pom:0.8' in repository ooo-repo (http://threerings.github.com/maven-repo)
[artifact:dependencies] Downloading: org/osflash/signals/0.8/signals-0.8.pom from repository flexlib-repo at http://maven.ow2.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'org.osflash:signals:pom:0.8' in repository flexlib-repo (http://maven.ow2.org/maven2)
[artifact:dependencies] Downloading: org/osflash/signals/0.8/signals-0.8.pom from repository ooo-maven-repo at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'org.osflash:signals:pom:0.8' in repository ooo-maven-repo (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: org/osflash/signals/0.8/signals-0.8.pom from repository ooo-external at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'org.osflash:signals:pom:0.8' in repository ooo-external (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: org/osflash/signals/0.8/signals-0.8.pom from repository central at http://repo1.maven.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'org.osflash:signals:pom:0.8' in repository central (http://repo1.maven.org/maven2)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.3/naryalib-1.3.pom from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.3' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.3/naryalib-1.3.pom from repository flexlib-repo at http://maven.ow2.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.3' in repository flexlib-repo (http://maven.ow2.org/maven2)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.3/naryalib-1.3.pom from repository ooo-maven-repo at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.3' in repository ooo-maven-repo (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.3/naryalib-1.3.pom from repository central at http://repo1.maven.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.3' in repository central (http://repo1.maven.org/maven2)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.2/naryalib-1.2.pom from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.2' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.2/naryalib-1.2.pom from repository flexlib-repo at http://maven.ow2.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.2' in repository flexlib-repo (http://maven.ow2.org/maven2)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.2/naryalib-1.2.pom from repository ooo-maven-repo at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.2' in repository ooo-maven-repo (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/naryalib/1.2/naryalib-1.2.pom from repository ooo-repo at http://build.earth.threerings.net/maven-repository
[artifact:dependencies] Error transferring file: build.earth.threerings.net
[artifact:dependencies] [WARNING] Unable to get resource 'com.threerings:naryalib:pom:1.2' from repository ooo-repo (http://build.earth.threerings.net/maven-repository): Error transferring file: build.earth.threerings.net
[artifact:dependencies] Downloading: com/threerings/naryalib/1.2/naryalib-1.2.pom from repository central at http://repo1.maven.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:naryalib:pom:1.2' in repository central (http://repo1.maven.org/maven2)
[artifact:dependencies] [INFO] snapshot com.threerings:whirledlib:1.1-SNAPSHOT: checking for updates from ooo
[artifact:dependencies] [WARNING] repository metadata for: 'snapshot com.threerings:whirledlib:1.1-SNAPSHOT' could not be retrieved from repository: ooo due to an error: Error transferring file: build.earth.threerings.net
[artifact:dependencies] [INFO] Repository 'ooo' will be blacklisted
[artifact:dependencies] Downloading: com/threerings/whirledlib/1.1-SNAPSHOT/whirledlib-1.1-SNAPSHOT.pom from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirledlib:pom:1.1-SNAPSHOT' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/whirledlib/1.1-SNAPSHOT/whirledlib-1.1-SNAPSHOT.pom from repository as3corelib-repo at https://github.com/samskivert/as3corelib/raw/master/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirledlib:pom:1.1-SNAPSHOT' in repository as3corelib-repo (https://github.com/samskivert/as3corelib/raw/master/repository)
[artifact:dependencies] Downloading: com/threerings/whirled-api-parent/1.1-SNAPSHOT/whirled-api-parent-1.1-SNAPSHOT.pom from repository flexmojos at http://repository.sonatype.org/content/groups/flexgroup/
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirled-api-parent:pom:1.1-SNAPSHOT' in repository flexmojos (http://repository.sonatype.org/content/groups/flexgroup/)
[artifact:dependencies] Downloading: com/threerings/whirled-api-parent/1.1-SNAPSHOT/whirled-api-parent-1.1-SNAPSHOT.pom from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirled-api-parent:pom:1.1-SNAPSHOT' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/whirled-api-parent/1.1-SNAPSHOT/whirled-api-parent-1.1-SNAPSHOT.pom from repository as3corelib-repo at https://github.com/samskivert/as3corelib/raw/master/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirled-api-parent:pom:1.1-SNAPSHOT' in repository as3corelib-repo (https://github.com/samskivert/as3corelib/raw/master/repository)
[artifact:dependencies] Downloading: com/threerings/vilyalib/1.0/vilyalib-1.0.pom from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:vilyalib:pom:1.0' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
[artifact:dependencies] Downloading: com/threerings/vilyalib/1.0/vilyalib-1.0.pom from repository as3corelib-repo at https://github.com/samskivert/as3corelib/raw/master/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:vilyalib:pom:1.0' in repository as3corelib-repo (https://github.com/samskivert/as3corelib/raw/master/repository)
[artifact:dependencies] Downloading: com/threerings/vilyalib/1.0/vilyalib-1.0.pom from repository flexmojos at http://repository.sonatype.org/content/groups/flexgroup/
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:vilyalib:pom:1.0' in repository flexmojos (http://repository.sonatype.org/content/groups/flexgroup/)
[artifact:dependencies] Downloading: com/threerings/vilyalib/1.0/vilyalib-1.0.pom from repository central at http://repo1.maven.org/maven2
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:vilyalib:pom:1.0' in repository central (http://repo1.maven.org/maven2)
[artifact:dependencies] Downloading: com/threerings/whirledlib/1.1-SNAPSHOT/whirledlib-1.1-SNAPSHOT.swc from repository ooo-public at http://ooo-maven.googlecode.com/hg/repository
[artifact:dependencies] Unable to locate resource in repository
[artifact:dependencies] [INFO] Unable to find resource 'com.threerings:whirledlib:swc:1.1-SNAPSHOT' in repository ooo-public (http://ooo-maven.googlecode.com/hg/repository)
     [copy] Copying 14 files to /home/whirledtest/Desktop/msoy/dist/lib

jprepare:
     [copy] Copying 1 file to /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/data/all

qcompile:
     [echo] Compiling main classes...
    [javac] Compiling 1048 source files to /home/whirledtest/Desktop/msoy/dist/classes
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/PropertySpaceMarshaller.java:35: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, SET_PROPERTY, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/ContentMarshaller.java:35: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, CONSUME_ITEM_PACK, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/ContentMarshaller.java:48: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, PURCHASE_ITEM_PACK, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/PrizeMarshaller.java:35: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, AWARD_PRIZE, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/PrizeMarshaller.java:48: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, AWARD_TROPHY, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMessageMarshaller.java:35: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, SEND_MESSAGE, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMessageMarshaller.java:48: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, SEND_PRIVATE_MESSAGE, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:35: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, ADD_TO_COLLECTION, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:48: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, CHECK_DICTIONARY_WORD, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:61: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, END_GAME_WITH_SCORES, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:74: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, END_GAME_WITH_WINNERS, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:87: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, END_ROUND, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:100: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, END_TURN, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:113: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, GET_COOKIE, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:126: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, GET_DICTIONARY_LETTER_SET, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:139: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, GET_DICTIONARY_WORDS, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:152: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, GET_FROM_COLLECTION, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:165: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, MAKE_PLAYER_AI, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:178: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, MERGE_COLLECTION, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:191: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, RESTART_GAME_IN, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:204: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, SET_COOKIE, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/WhirledGameMarshaller.java:217: warning: [deprecation] sendRequest(com.threerings.presents.client.Client,int,java.lang.Object[]) in com.threerings.presents.data.InvocationMarshaller has been deprecated
    [javac]         sendRequest(arg1, SET_TICKER, new Object[] {
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/WhirledGameManager.java:1038: warning: [deprecation] Interval() in com.samskivert.util.Interval has been deprecated
    [javac]         protected Interval _interval = new Interval() {
    [javac]                                        ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/WhirledGameManager.java:1038: warning: [deprecation] Interval() in com.samskivert.util.Interval has been deprecated
    [javac]         protected Interval _interval = new Interval() {
    [javac]                                                       ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/client/GameObjectImpl.java:117: cannot find symbol
    [javac] symbol  : method contains(java.lang.Object)
    [javac] location: class com.samskivert.util.ObserverList<java.lang.Object>
    [javac]             if (!_listeners.contains(obj)) {
    [javac]                            ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/client/GameObjectImpl.java:524: warning: [deprecation] SAFE_IN_ORDER_NOTIFY in com.samskivert.util.ObserverList has been deprecated
    [javac]         new ObserverList<Object>(ObserverList.SAFE_IN_ORDER_NOTIFY);
    [javac]                                              ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/client/GameObjectImpl.java:524: com.samskivert.util.ObserverList is abstract; cannot be instantiated
    [javac]         new ObserverList<Object>(ObserverList.SAFE_IN_ORDER_NOTIFY);
    [javac]         ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/data/PropertySetEvent.java:23: cannot find symbol
    [javac] symbol  : constructor NamedEvent()
    [javac] location: class com.threerings.presents.dobj.NamedEvent
    [javac]     {
    [javac]     ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/ContentDispatcher.java:21: type parameter com.whirled.game.data.ContentMarshaller is not within its bound
    [javac] public class ContentDispatcher extends InvocationDispatcher<ContentMarshaller>
    [javac]                                                             ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/PrizeDispatcher.java:21: type parameter com.whirled.game.data.PrizeMarshaller is not within its bound
    [javac] public class PrizeDispatcher extends InvocationDispatcher<PrizeMarshaller>
    [javac]                                                           ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/PropertySpaceDispatcher.java:21: type parameter com.whirled.game.data.PropertySpaceMarshaller is not within its bound
    [javac] public class PropertySpaceDispatcher extends InvocationDispatcher<PropertySpaceMarshaller>
    [javac]                                                                   ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/WhirledGameDispatcher.java:21: type parameter com.whirled.game.data.WhirledGameMarshaller is not within its bound
    [javac] public class WhirledGameDispatcher extends InvocationDispatcher<WhirledGameMarshaller>
    [javac]                                                                 ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/game/server/WhirledGameMessageDispatcher.java:21: type parameter com.whirled.game.data.WhirledGameMessageMarshaller is not within its bound
    [javac] public class WhirledGameMessageDispatcher extends InvocationDispatcher<WhirledGameMessageMarshaller>
    [javac]                                                                        ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/remix/data/EditableDataPack.java:306: warning: [deprecation] FAST_UNSAFE_NOTIFY in com.samskivert.util.ObserverList has been deprecated
    [javac]         new ObserverList<ChangeListener>(ObserverList.FAST_UNSAFE_NOTIFY);
    [javac]                                                      ^
    [javac] /home/whirledtest/Desktop/msoy/src/java/com/threerings/msoy/whirled-api/core/src/main/java/com/whirled/remix/data/EditableDataPack.java:306: com.samskivert.util.ObserverList is abstract; cannot be instantiated
    [javac]         new ObserverList<ChangeListener>(ObserverList.FAST_UNSAFE_NOTIFY);
    [javac]         ^
    [javac] 9 errors
    [javac] 26 warnings

BUILD FAILED
/home/whirledtest/Desktop/msoy/build.xml:911: Compile failed; see the compiler error output for details.

Total time: 31 seconds

We pasted the whirled-api folder into the src>java>com>threerings>msoy folder. Then we did a ant distclean to clean the files and re ran ant distall.

(As far as we are going, me and shadowsych are working together to make a whirled instance to work.)

I would also refer to this 100 error version that we had asked before. https://github.com/greyhavens/msoy/issues/20

We also love to see a quick reply if you can.

**Comment by Shadowsych** [comment]
I went over to the GitHub for the Whirled-API, and inside the directories for where the coding problems occurred, and I saw this:

**"The code in here is out of date and out of sync, but will not be updated
until needed, as we're focusing on flash development."**

(https://github.com/greyhavens/whirled-api/tree/master/core/src/main/java/com/whirled/game)

If this problem is related to the outdated code/out of sync code, I think this is a time when it is needed to be fixed... If not, please guide us through what we should do. 

Currently we have the copy of the Whirled-API inside:  /msoy/src/java/com/threerings/directory 

Thanks!

**Comment by Shadowsych** [comment]
We got it to work by not doing the distclean command and then the distall command. I guess I didn't read that the distclean command cleans out the compiled code...

How we fixed it:
1. Removed the Whirled-API from this directory /msoy/src/java/com/threerings/directory
2. Instead input the Whirled-Code.jar into that direcotry /msoy/src/java/com/threerings/directory
3. DO NOT distclean, straight up do a distall and the compiler should finish the code

The compilation has been fixed now, we are currently on an error with the 'server_url' variable. If we need help on this current error, we'll put up a github post.

**Comment by Shadowsych** [comment]
We fixed the "server_url" variable error because the msoy-build.xml had to find our flex directories. We also had to change the flex directories to many other places and such in the msoy-build.xml. 

We are using the Flex 3.0 SDK.

Currently we're receiving an actionscript 3.0 error stating:

Call to a possibly undefined method activate through a reference with static type mx.managers:ISystemManager.

We will see how we can fix it.
